src/sage/modular/modform/ring.py

```diff
@@ -348,9 +347,9 @@ def polynomial_ring(self, names, gens=None):
           generators returned by the method
           :meth:`~sage.modular.modform.find_generator.ModularFormsRing.gen_forms`
           is used instead. Note that we do not check if the list is
-          indeed a generating set.
+          indeed a generating set
 
```

Comment:
The "rule" is: If the input description is a single sentence, omit the period at the end. If it consists of several full sentences, each of them ends with a period.

Hence please a period here.